interfaces/adfsuite/ams.py

```diff
@@ -2297,12 +2346,21 @@ def get_runscript(self):
             ret += "\n"
         return ret
 
-    def check(self):
+    def check(self) -> bool:
         """Check if ``termination status`` variable from ``General`` section of main KF file equals ``NORMAL TERMINATION``."""
         try:
             status = self.results.readrkf("General", "termination status")
+        except (FileError, KeyError) as e:
+            log(str(e), 1)
+            return False
         except:
```

Comment:
Unrelated to this PR, but since we are touching the method already: I think the bare except will catch things like ``KeyboardInterrupt`` and should generally be avoided.

Reply:
True... I am not going to change this here though, as I don't have a feel for which other errors are currently being caught by the bare except which could break the check behaviour if there is a common one